src/api/hocrrenderer.cpp

```diff
@@ -228,12 +231,26 @@ char *TessBaseAPI::GetHOCRText(ETEXT_DESC *monitor, int page_number) {
         case PT_CAPTION_TEXT:
           hocr_str << "ocr_caption";
           break;
+        case PT_FLOWING_IMAGE:
+        case PT_HEADING_IMAGE:
+        case PT_PULLOUT_IMAGE:
+          {
+            if(hocr_images){
+              hocr_str << "ocr_photo";
+              skipword = true;
+            }
+            break;
```

Comment:
The code will never fall through (as the comment claims) because of this `break` statement.

Reply:
My bad, I thought I had already moved that break statement to inside the if. Will address that now.